### ISSUE TITLE
ffmpeg to version 7.1

### ISSUE THREAD

**Issue by msharp1** [getsolus/packages#4191]
### Please confirm there isn't already an existing package update request

- [X] I have searched through package update requests

### Name

ffmpeg

### Version

7.1

### Link to the release notes or announcement.

https://ffmpeg.org/index.html#news

### Reason for updating

The 7.x branch implements `DVD-Video demuxer, powered by libdvdnav and libdvdread.` which can now ingest DVDs directly from block device (cool !).

[Link to the doc](https://ffmpeg.org/ffmpeg-formats.html#toc-dvdvideo)

**Comment by msharp1** [comment]
Might be useful to circumvent https://discuss.getsol.us/d/11070-handbrake-with-amd-hardware-encoding-support-vcn/4

**Comment by malfisya** [comment]
This will happen and the staffs are aware, but it is not going to be soon. Updating underlying library like `ffmpeg` involved rebuilding packages that depends on it, [last time](https://github.com/getsolus/packages/pull/1380/commits) it involved 74 packages. We also have to wait until all those packages support `ffmpeg` 7.1. It can be a while.

**Comment by androidnisse** [comment]
Resolved in https://github.com/getsolus/packages/pull/4362

**Comment by malfisya** [comment]
> Thank you ! However I think it's not configured with `--enable-libdvdnav` and `--enable-libdvdread`. Do I need to open a new issue for that ?

Yes. Explain the reason why that flag is needed on the new issue.
